exasol_integration_test_docker_environment/lib/api/spawn_test_environment_with_test_container.py

```diff
@@ -36,8 +37,7 @@ def _cleanup(environment_info: EnvironmentInfo) -> None:
 def spawn_test_environment_with_test_container(
         environment_name: str,
         test_container_content: TestContainerContentDescription,
-        database_port_forward: Optional[int] = None,
-        bucketfs_port_forward: Optional[int] = None,
+        port_forwarding: Optional[PortForwarding] = None,
```

Comment:
hm, now the two API calls for spawn_test_environmen diverge in the same parameters, not a fan off, and this API change would require a major version release, the API has user contact

Reply:
Which two API calls do you mean? These two?
* `api/spawn_test_environment_with_test_container.py`: `spawn_test_environment_with_test_container()`
* `api/spawn_test_environment.py`: `spawn_test_environment()`

If true, then the latter I was not sure if we can change, as I assume a CLI parameter cannot be mapped to an instance of class `PortForwarding`, please correct me if I'm wrong. 

So if you prefer we can change the signature of the first API call back to single arguments of basic types like `int` and `str`.

Reply:
the main question, we need to answer, is the assumption of the cli - api compatibility tests really true, that they need to reflect each other. To be honest, I have more and more doubts about this. Especially, regarding the default values. However, stays the question with what do we replace the compatibility tests